modules/ingest-geoip/build.gradle

```diff
@@ -39,6 +39,14 @@ restResources {
   }
 }
 
+compileJava {
+  options.compilerArgs << '-Xlint:-classfile'
+}
+
+compileTestJava {
+  options.compilerArgs << '-Xlint:-classfile'
+}
+
```

Comment:
For now, suppress the lint warnings to avoid issues like this:

```
/Users/chegar/.gradle/caches/modules-2/files-
2.1/com.maxmind.geoip2/geoip2/3.0.0/523b0d63f3dbeacb7dfceb7431cb17fa56cf79fa/geoip2-
3.0.0.jar(/com/maxmind/geoip2/model/CityResponse.class): 
warning: Cannot find annotation method 'value()' in type 'JsonProperty': class file for 
com.fasterxml.jackson.annotation.JsonProperty not found
```

There is likely a javac issue here. Investigating.

Reply:
@breskeby Is there a better way to suppress the warning here, rather than three separate blocks?